script/server

```diff
@@ -7,123 +7,25 @@ set -e
 
 cd "$(dirname "$0")/.."
 
-echo "==> Updating..."
-script/update
-
-AWS_PROFILE=${AWS_PROFILE:-dalmatian-admin}
-
-if [ "$(command -v dalmatian)" ]
+if ! [[ "$(command -v dalmatian)" ]]
```

Comment:
It seemed like a good idea to keep the basic check to make sure you've followed the readme and downloaded Dalmatian Tools.